pkg/search/postgres/store.go

```diff
@@ -95,14 +97,24 @@ func NewGenericStoreWithPermissionChecker[T any, PT unmarshaler[T]](
 	checker walker.PermissionChecker,
 ) *GenericStore[T, PT] {
 	return &GenericStore[T, PT]{
-		db:                               db,
-		schema:                           schema,
-		pkGetter:                         pkGetter,
-		copyFromObj:                      copyFromObj,
-		insertInto:                       insertInto,
-		setAcquireDBConnDuration:         setAcquireDBConnDuration,
-		setPostgresOperationDurationTime: setPostgresOperationDurationTime,
-		permissionChecker:                checker,
+		db:          db,
+		schema:      schema,
+		pkGetter:    pkGetter,
+		copyFromObj: copyFromObj,
+		insertInto:  insertInto,
+		setAcquireDBConnDuration: func() durationTimeSetter {
```

Comment:
What's the reason behind this change? When can `setAcquireDBConnDuration` be null?

Reply:
We do not log metrics in migrations, but we copy store code that has metric logging.